src/components/TodosLogic.jsx

```diff
@@ -31,6 +32,7 @@ const TodosLogic = () => {
 
   const delTodo = (id) => {
     setTodos([
+      // eslint-disable-next-line arrow-body-style
```

Comment:
It actually work well but please let work without disabling our linters for good code practices